Autopilot/AttitudeManager/Attitude_States.hpp

```diff
@@ -0,0 +1,14 @@
+/**
+ *  Attitude State Machine Header
+ */
+#ifndef ATTITUDESTATES_HPP
+#define ATTITUDESTATES_HPP
+
+//Using static so we don't have to create an instance of Attitude States
```

Comment:
not what static means. This would be true if we were dealing with a c++ class.
Having a stand alone function be static means its scope is only limited to its object file. No one outside your module would be able to call these functions; not what you want.

On a side note, using static is a great way to make a global variable file scope or to make helper functions that you have no interest in any other modules knowing about private to that module.